config/vufind/EDS.ini

```diff
@@ -323,3 +323,27 @@ AU = None
 ; documentation on options for different adapters.
 ; ttl, Time to Live, i.e. cache entry life time in seconds. 300 seconds by default.
 ;options[ttl] = 300
+
+; This section supports the Databases recommendation module,
+; which displays a list of the results' databases, each linking
+; to that database's individual website.
+[Databases]
```

Comment:
With the generalization beyond EDS.ini, this boilerplate could technically be in any backend .ini file.  But this still seems reasonable?

Reply:
The recommendation module itself is documented in searches.ini. If we keep these settings here, we should make sure that documentation has a reference to this so it can be found. I think the searches.ini documentation also needs to be updated to include the setting to specify the config file.

Reply:
Thanks, I updated the seraches.ini description to include that parameter and reference EDS.ini.

Reply:
Yup searches.ini documentation does include the .ini file setting and references the example in EDS.ini.